docs/intro.md

```diff
@@ -1,29 +1,38 @@
 ---
+title: Introduction
+hide_title: false
+draft: false
+sidebar_label: Introduction
 sidebar_position: 0
+keywords:
+  - deriv-api
+  - getting-started-deriv-api
```

Comment:
If we will implement a search feature, I assume that it will also be able to find based on keywords. Then I think it is better to create a list of single keywords to make it easier for the search engine to get search queries.

```
keywords:
- deriv
- api
- getting
- started
```